src/components/tab-bar/tab-bar.scss

```diff
@@ -35,6 +35,15 @@ $tab-scroller-fade-width: 65;
     }
 }
 
+.mdc-tab--active {
+    .mdc-ripple-upgraded--background-focused {
+        &:before {
+            background-color: transparent;
+            transition: background-color 1s ease;
```

Comment:
When the tab-bar is first loaded, hovering over the selected tab gives a hover effect. Once any tab has been clicked on, hovering over the selected tab no longer gives that hover effect.

https://user-images.githubusercontent.com/154725/114146639-22f9e780-9918-11eb-9517-cd06465f37d2.mov

That in itself isn't much of an issue I think, but when the currently selected tab receives focus by keyboard navigation, it highlights only to immediately fade out again. I'm not sure if that's going to be difficult to fix? I leave it to you to determine if we can merge this anyway, or if a fix is needed first.

Reply:
https://user-images.githubusercontent.com/154725/114147492-0b6f2e80-9919-11eb-8a6f-4acbe8d3e101.mov

Reply:
Thanks. First one is fine I'd say. Displaying a hovering effect on an element which is already active and doesn't change even if you click on it is quite unnecessary (it's basically non-interactive in that state, so doesn't need a visual feedback on hover). But if it behaves in a certain way when the tab-bar is first loaded, hmm... is not even worth thinking about I'd say. 😄 it doesn't hurt to show that effect.

For the second point, I was fiddling around for a while to make a simple fix with minimal CSS. I think the effect is fine for keyboard navigation. Just to quickly "mention" that you just jumped to this tab (but it's already active).

In this video I'm playing with arrow keys back and forth. 


https://user-images.githubusercontent.com/35954987/114173580-ae36a580-9937-11eb-8f24-86474ad03e92.mp4

This effect is intentional and I personally like it. I don't think it requires any fix. Don't you like it?

Reply:
Good enough for me 🙂